http/headers/Expect-CT.json

```diff
@@ -23,9 +25,7 @@
               "version_added": false
             },
             "oculus": "mirror",
-            "opera": {
-              "version_added": "48"
-            },
+            "opera": "mirror",
```

Comment:
This causes the `notes` from Chrome to be mirrored, but this is probably fine, even if the "later builds of" part might not apply.